src/js/App/GlobalFilter/GlobalFilter.js

```diff
@@ -23,19 +136,22 @@ const GlobalFilter = () => {
   const [token, setToken] = useState();
   const dispatch = useDispatch();
   const { isLoaded, count, total, sapCount, isDisabled } = useSelector(
-    ({ globalFilter: { tags, sid, workloads, globalFilterHidden }, chrome: { appId } }) => ({
-      isLoaded: tags?.isLoaded && sid?.isLoaded && workloads?.isLoaded,
-      count: tags?.count || 0 + sid?.count || 0 + workloads?.count || 0,
-      total: tags?.total || 0 + sid?.total || 0 + workloads?.total || 0,
-      sapCount: workloads?.hasSap,
-      isDisabled: globalFilterHidden || !appId,
-    }),
+    ({ globalFilter: { tags, sid, workloads, globalFilterHidden }, chrome: { appId } }) => {
```

Comment:
According to @karelhala chrome can be empty and we should add safe accessors.

Reply:
Good point, altough I would let this pass since the global filter can't be visible on unauth pages. But better be safe than sorry.

Reply:
This actually can't happen because the global filter is not visible when chrome is `undefined`. The visibility of the global filter depends on `chrome.activeLocation`. If chrome is undefined, the global filter will not be rendered.